charts/cinder-csi-plugin/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 appVersion: latest
```

Comment:
can u change the appVersion to v1.19 ? latest can be kept for only master branch